### PR TITLE
build.d: Add style

### PR DESCRIPTION
**Pull request by MoonlightSentinel**

### CODE REVIEW

**Comment by thewilsonator** [comment]
Needs a rebase.

**Comment by MoonlightSentinel** [comment]
@thewilsonator Done

**Review by thewilsonator — APPROVED** [review]

File: src/build.d [hunk 1]
```diff
@@ -431,6 +432,45 @@ alias checkwhitespace = makeDep!((builder, dep) => builder
     })
 );
 
+alias style = makeDep!((builder, dep)
+{
+    const dir = env["G"].buildPath("dscanner");
```
Comment (marler8997) [inline src/build.d]: How about `scannerDir`?  `dir` seems a bit to generic

Comment (MoonlightSentinel) [reply src/build.d]: Done

**Review by marler8997 — APPROVED** [review]

**Comment by thewilsonator** [comment]
Auto-merge toggled on

**Pull request merged**